test/dotnet-publish.Tests/PublishAppWithBuildDependency.cs

```diff
@@ -17,8 +17,6 @@ public void PublishExcludesBuildDependencies()
             var testInstance = TestAssetsManager.CreateTestInstance("AppWithDirectDependencyAndTypeBuild")
                 .WithLockFiles();
 
-            new RestoreCommand() { WorkingDirectory = testInstance.TestRoot }.Execute().Should().Pass();
```

Comment:
@Sridhar-MS I wonder if the lack of custom args is intentional, might be worth finding out who wrote the test